tools/generate_kopernicus.cpp

```diff
@@ -206,6 +225,25 @@ void GenerateKopernicusForSlippist1(
     kopernicus_cfg << "  }\n";
   }
   kopernicus_cfg << "}\n";
+  for (std::string const& name : solar_system.names()) {
+    bool const is_star =
+        !solar_system.keplerian_initial_state_message(name).has_parent();
+    if (is_star) {
```

Comment:
Preference for `if (!is_star) { ... }`. `continue` is a polite spelling of `goto`.